asciidocs/setup.adoc

```diff
@@ -111,15 +114,15 @@ You should see something similar to the following:
 
 image::gitpod.welcome.png[]
 
-**The sidebar** allows you to customise your Gitpod environment and perform basic tasks (Copy/Paste, Open files, search, git, etc.) Click the Explorer button to see which files are in this repository.
+**The sidebar** allows you to customise your Gitpod environment and perform basic tasks (copy, paste, open files, search, git, etc.). Click the Explorer button to see which files are in this repository.
 
 **The terminal** allows you to run all the programs in the repository, for example `nextflow` and `docker` are installed. 
 
 **The main window** allows you to view and edit files. Clicking on a file in the explorer will open it within the main window. We can also see the nf-training material in this window, but if you find the space too small you can open it separately in another browser window (https://training.seqera.io/). Finally, if you accidently close the Gitpod training window, you can reopen it by entering `gp preview https://training.seqera.io`.
 
-4. Saving your files in Gitpod to local machine.
+4. Saving your files in Gitpod to your local machine.
 
-To save your files, choose your file of interest, then either use the left/top hand side bar (File/Save As...) or use you're keyboard shortcut to save as (On Mac: CMD, shift, s), then choose `Show local`. This will open up a explorer to choose where to save your file on your local machine.
+To save your files, choose your file of interest, then either use the left/top hand side bar (File/Save As...) or use your keyboard shortcut to save as (On Mac: CMD, shift, s), then choose `Show local`. This will open up a explorer to choose where to save your file on your local machine.
```

Comment:
```suggestion
To save your files, choose your file of interest, then either use the left/top hand sidebar (File/Save As...) or use your keyboard shortcut to save as (On Mac: CMD, shift, s), then choose `Show local`. This will open up an explorer to choose where to save your file on your local machine.
```